src/test/java/com/puppycrawl/tools/checkstyle/checks/naming/ConstantNameCheckTest.java

```diff
@@ -86,7 +86,7 @@ public void testDefault()
 
         final String[] expected = {
             "25:29: " + getCheckMessage(MSG_INVALID_PATTERN, "badConstant", pattern),
-            "142:30: " + getCheckMessage(MSG_INVALID_PATTERN, "BAD__NAME", pattern),
+            "139:30: " + getCheckMessage(MSG_INVALID_PATTERN, "BAD__NAME", pattern),
```

Comment:
We are not expecting changes to this file.
Changes for your issue should be in an input file that no other tests uses to avoid this conflict.